universal-application-tool-0.0.1/conf/messages

```diff
@@ -23,75 +34,113 @@ button.deleteEntity=Delete {0}
+toast.applicationSaved=Successfully saved application: application ID {0}
+
+# A toast message that displays when an applicant tries to apply to a previously completed program.
+toast.programCompleted=Application was already completed.
 
 #----------------------------------------------------------#
-# QUESTIONS - contains text specific to viewing questions. #
+# ADDRESS QUESTION - text when viewing an address question #
 #----------------------------------------------------------#
 
-# Address question labels and placeholder text
-label.street=Address line 1
-placeholder.street=Street address
+# Address question labels - these are shown as the field label before an input box.
 label.addressLine2=Address line 2
-placeholder.line2=Apartment, suite, unit, building, floor, etc.
 label.city=City
-placeholder.city=City
 label.state=State
-placeholder.state=State
+label.street=Address line 1
 label.zipcode=ZIP Code
-placeholder.zipcode=ZIP Code
-
-# Name question labels and placeholder text
-label.firstName=First name
-placeholder.firstName=First name
-label.middleName=Middle name
-placeholder.middleName=Middle name
-label.lastName=Last name
-placeholder.lastName=Last name
 
-# Enumerator question text
-# Text on a button an applicant would click to add another entity
-button.addEnumeratorEntity=add element
+# Question placeholders - these are shown in the text input boxes, before a user enters an answer.
+placeholder.city=City
+placeholder.line2=Apartment, suite, unit, building, floor, etc.
+placeholder.state=State
+placeholder.street=Street address
+placeholder.zipcode=ZIP Code
 
-# Question validation
-validation.firstNameRequired=Please enter your first name.
-validation.lastNameRequired=Please enter your last name.
-validation.textTooShort=Must contain at least {0} characters.
-validation.textTooLong=Must contain at most {0} characters.
-validation.numberTooSmall=Must be at least {0}.
-validation.numberTooBig=Must be at most {0}.
-validation.tooFewSelections=Please select at least {0}.
-validation.tooManySelections=Please select fewer than {0}.
+# Validation errors that are shown when a user enters an invalid address.
 validation.streetRequired=Please enter valid street name and number.
 validation.cityRequired=Please enter city.
 validation.stateRequired=Please enter state.
 validation.zipcodeRequired=Please enter valid ZIP code.
 validation.invalidZipcode=Please enter valid 5-digit ZIP code.
 validation.noPoBox=Please enter a valid address. We do not accept PO Boxes.
+
+#--------------------------------------------------------------------------------------------------------#
+# ENUMERATION QUESTION - text when viewing a question where a user must name entities, such as children. #
+#--------------------------------------------------------------------------------------------------------#
+
+# Text on a button an applicant would click to add another entity
+button.addEnumeratorEntity=add element
+
+button.deleteEntity=Delete {0}
```

Comment:
(non-blocking): should this have `ariaLabel` in the name somewhere since that's how its used?

Reply:
Great catch, thank you! Fixed